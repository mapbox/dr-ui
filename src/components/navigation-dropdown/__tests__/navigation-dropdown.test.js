import renderer from 'react-test-renderer';
import { testCases } from './navigation-dropdown-test-cases.js';

describe('navigation-dropdown', () => {
  describe(testCases.basic.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.basic;
      wrapper = renderer.create(testCase.element);
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
