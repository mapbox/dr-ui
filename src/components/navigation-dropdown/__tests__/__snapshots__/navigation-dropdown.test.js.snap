// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigation-dropdown Basic renders as expected 1`] = `
<div
  className="relative "
>
  <div
    className="select-container "
  >
    <select
      aria-required={true}
      className="select select--stroke round-full bg-white"
      data-test="navigate-this-section-select"
      disabled={false}
      id="navigate-this-section"
      onChange={[Function]}
      value="page-one"
    >
      <option
        value="page-one"
      >
        Title one
      </option>
      <option
        value="page-two"
      >
        Title two
      </option>
    </select>
    <div
      className="select-arrow"
    />
  </div>
  <div
    role="alert"
  />
</div>
`;
